src/GrumPHP/Task/Atoum.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Atoum task
+ */
+class Atoum extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'atoum';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults(array(
+            'config_file' => null,
+            'bootstrap_file' => null,
+            'directories' => array(),
+            'files' => array(),
+            'namespaces' => array(),
+            'methods' => array(),
+            'tags' => array(),
+        ));
+
+        $resolver->addAllowedTypes('config_file', array('null', 'string'));
+        $resolver->addAllowedTypes('bootstrap_file', array('null', 'string'));
+        $resolver->addAllowedTypes('directories', array('array'));
+        $resolver->addAllowedTypes('files', array('array'));
+        $resolver->addAllowedTypes('namespaces', array('array'));
+        $resolver->addAllowedTypes('methods', array('array'));
+        $resolver->addAllowedTypes('tags', array('array'));
+
+        return $resolver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return ($context instanceof GitPreCommitContext || $context instanceof RunContext);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function run(ContextInterface $context)
+    {
+        $files = $context->getFiles()->name('*.php');
```

Comment:
Yes, Atoum allows to run tests for specific files but `$files` would have to be the test files. AFAIK, if a file that is unit tested is given to Atoum command line, it won't know which tests to execute. So i'm not sure it would be very usefull to use.